sonarqube/resource_sonarqube_qualitygate.go

```diff
@@ -60,23 +60,22 @@ func resourceSonarqubeQualityGate() *schema.Resource {
 			"name": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
 			},
 			"copy_from": {
-				Type:     schema.TypeString,
-				Optional: true,
-				ForceNew: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"condition"},
 			},
 			"is_default": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Description: "Name of the quality gate to set as default",
+				Description: "When set to true this Quality Gate is set as default",
 				Default:     false,
-				ForceNew:    true,
 			},
-			"conditions": {
+			"condition": {
 				Type:        schema.TypeList,
-				Computed:    true,
+				Optional:    true,
```

Comment:
condition is a required field now right?

Reply:
No because of `copy_from` - you must specify one or the other but you can’t have both